sentry/src/main/java/io/sentry/transport/RateLimiter.java

```diff
@@ -203,7 +203,10 @@ public void updateRetryAfterLimits(
               for (final String catItem : categories) {
                 DataCategory dataCategory = DataCategory.Unknown;
                 try {
-                  dataCategory = DataCategory.valueOf(StringUtils.capitalize(catItem));
+                  final String catItemCapitalized = StringUtils.capitalize(catItem);
+                  if (catItemCapitalized != null) {
+                    dataCategory = DataCategory.valueOf(catItemCapitalized);
+                  }
```

Comment:
```suggestion
                  }
                  else {
                    logger.log(ERROR, "Couldn't capitalize: %s", catItem);
                  }
```

Also, looks like like 211 should be ERROR, no?

Reply:
makes sense